src/index.ts

```diff
@@ -53,11 +123,29 @@ export const getRequestBody = (
 
   requestBody['pd[tm]'] = payload.tm || 'pinterest-mc'
 
+  // match  event types to Pinterest's default
+
+  if (ecommerce) {
+    switch (eventType) {
```

Comment:
A more concise way to do this might be to store an object const that maps the event Type to the event name, like

```ts
const eventTypes = {
  "Product Added": "addtocart",
  "Order Completed": "checkout",
}

/// and then

if (eventTypes[eventType]) {
  request.body.event = eventTypes[eventType]
}
```